### PR TITLE
chore: Bump kommander chart version

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**
chore

**What this PR does/ why we need it**:
bumps kommander to latest version https://github.com/mesosphere/charts/pull/790

**Which issue(s) this PR fixes**:

no issue
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
no issue
```

**Checklist**

- [x] The commit message explains the changes and why are needed.
- [ ] The code builds and passes lint/style checks locally.
- [ ] The relevant subset of integration tests pass locally.
- [ ] The core changes are covered by tests.
- [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by GoelDeepak — APPROVED** [review]

**Comment by gracedo** [comment]
`pod kubecost-checker-grnlb in namespace default has reached phase Failed` kubecost checks are failing but there are no artifacts so it's been hard troubleshooting - haven't been able to get a cluster up due to hanging `aws_iam_role.node_role: Still creating...` so I may need to wait till tomorrow to try to see what's happening here

**Comment by gracedo** [comment]
Finally got a cluster up, the aggregatedCostModel endpoint is now returning 500s (failing this check https://github.com/mesosphere/kubeaddons-kommander/blob/master/test/artifacts/kubecost-checker.yaml#L35-L40). I'm checking with kubecost team whether this is now expected with the update or if maybe they introduced a bug.

File: addons/kommander/1.2.0/kommander-12.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-11"
```
Comment (makkes) [inline addons/kommander/1.2.0/kommander-12.yaml:9]: I believe this has to be bumped as well.

File: addons/kommander/1.2.0/kommander-12.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-beta.2"
```
Comment (makkes) [inline addons/kommander/1.2.0/kommander-12.yaml:10]: Maybe this, too but @juliangieseke might know better.

File: addons/kommander/1.2.0/kommander-12.yaml (hunk 1 above)
Comment (juliangieseke) [reply addons/kommander/1.2.0/kommander-12.yaml:9]: this one needs to be bumped, yes.

File: addons/kommander/1.2.0/kommander-12.yaml (hunk 2 above)
Comment (juliangieseke) [reply addons/kommander/1.2.0/kommander-12.yaml:10]: this one not. we'll bump the appversion when doing released - nothing that needs to be done in regular PRs.

File: addons/kommander/1.2.0/kommander-12.yaml (hunk 1 above)
Comment (juliangieseke) [reply addons/kommander/1.2.0/kommander-12.yaml:9]:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-12"
```

**Review by juliangieseke — APPROVED** [review]

**Comment by shaneutt** [comment]
CI is reporting `harness.go:71: ERROR: kubecost checker job did not succeed within timeout` :thinking:

**Comment by gracedo** [comment]
Still in discussion with Kubecost team whether the new error message in UI are intentional or if we want to at least change the error message. One thing I do need to do is make sure the new `Savings` recording rule is included in the latest version so I will be closing this pr for now, making that fix in the chart, and reopening a pr once those chart fixes are done. Depending on outcome of conversations I may also need to modify our kubecost checker test script if we now expect the aggregatedCostModel endpoint to return 500s instead of 200s until clusters are attached

**Pull request closed without merging**